test/indexTest.js

```diff
@@ -0,0 +1,11 @@
+var assert = require("assert")
+var server = require("./testServer")
```

Comment:
@averyvery here's my shot at this. duplicating some server setup logic, if that changes a lot I might be in trouble with this implementation, but at least it keeps my test and development servers separate.